Source/ProjectFirma.Web/Models/GrantAllocationLikelyPerson.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ProjectFirma.Web.Models
+{
+    public partial class GrantAllocationLikelyPerson : IAuditableEntity
+    {
+        public string AuditDescriptionString => $"Grant Allocation Likely Person: {GrantAllocation?.DisplayName} - {Person?.FullNameFirstLast}";

```

Comment:
for AuditDescriptionStrings on lookup tables, I just use the FKs on the table instead of the display names. This works and provides more detail in logs if the full object is loaded

Reply:
no need to change, just providing another option

Reply:
Good idea, changing!